dev_tools/requirements/envs/dev.env.txt

```diff
@@ -10,6 +10,10 @@ accessible-pygments==0.0.4
     # via pydata-sphinx-theme
 alabaster==0.7.13
     # via sphinx
+appnope==0.1.3
```

Comment:
how hard would it be to re-generate from a linux box so we don't get this platform specific churn

Reply:
oh wait I forgot the docker thing is already merged. Please use that to generate these files (ping me if you need help or want me to do it)

Reply:
Regenerated using the docker script.